forwards_paper.tex

```diff
@@ -749,21 +775,35 @@ \subsection*{Tree sequence simplification}
 before and after simplification,
 and Figure~\ref{fig:simplify_state} depicts the ``paint pot'' state of the algorithm
 during the process of simplifying this tree sequence.
+Since the method begins with the samples and moves back through time,
+in the output tree sequence, the $n$ samples will be numbered $0, 1, \ldots, n-1$
+and subsequent nodes will be ordered by time since birth.
+This is seen in the red labels of Figure~\ref{fig:method_diagram}.
+
 
 \begin{figure}
     \begin{center}
         \includegraphics{simplify-state-diagram}
     \end{center}
     \caption{
         A depiction of the state of the simplification algorithm
-        at each point in time,
+        as it moves up through the embellished pedigree
```

Comment:
s/as it moves/after it has moved/

Reply:
Ok that's probably better as it is depicting both. (Also why it's tricky to understand =)